docs/corpora_schema.md

```diff
@@ -34,30 +35,44 @@ schema. If it does not, it will reject the dataset with an appropriate error mes
+    so we require that genes are present in both datasets. Summarizing, any cell barcodes that are removed from the data MUST be filtered from both raw and final matrices.
+    By contrast, genes MUST NOT be filtered from the raw matrix. Any genes that publishers wish to filter from the final matrix MAY have their values replaced by np.nan,
+    which will mask them from exploration.
+
+In addition to these general requirements, the following table describes the matrix data and layers requirements that are assay-specific. If cellxgene does not support
+an assay you would like to publish, please post an issue on this repository to start a conversation about extending the schema. If an entry in the table is empty,
+the cellxgene schema does not have any other requirements on data in those layers beyond the ones listed above. This is usually the case when there are many ways to
+produce the matrix layer in question.
+
+| Assay                                 | Raw requirements                                                                                                                              | Raw Required? | Final Requirements                                                                                 | Final Required?      | Other Layers |
+|---------------------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------|---------------|----------------------------------------------------------------------------------------------------|----------------------|--------------|
+| scRNA-seq (UMI, e.g. 10x v3)          | Values MUST be de-duplicated molecule counts.                                                                                                 | REQUIRED      |                                                                                                    | STRONGLY RECOMMENDED | OPTIONAL     |
+| scRNA-seq (non-UMI, e.g. SS2)         | Values MUST be one of read counts (e.g. FeatureCounts) or  estimated fragments (e.g. output of RSEM).                                         | REQUIRED      |                                                                                                    | STRONGLY RECOMMENDED | OPTIONAL     |
+| Accessibility (e.g. ATAC-seq, mC-seq) |                                                                                                                                               | NOT REQUIRED  | Values MUST correspond to gene features (e.g. HGNC symbols if human)                               | REQUIRED             | OPTIONAL     |
 
 ### Schema Version
 
-Datasets in the Data Portal must store the version of the schema they follow (that is, the version of this document) as
+Datasets in the Data Portal MUST store the version of the schema they follow (that is, the version of this document) as
```

Comment:
The link to the encoding is still broken. Pablo [found](https://chanzuckerbergteam.slack.com/archives/C018B64J3HN/p1617907599111700) the missing encoding. Do we really need a separate encoding field?

Also _and describes ~techincal~ technical details_

Reply:
I've replaced encoding link. I do not think we need a separate encoding, and in fact think it is confusing that we do, since we're standardizing on a single ingestion format. 

I think the integration will be significant and should be done in a separate PR.